src/System.Runtime.CompilerServices.Unsafe/tests/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.3;
+      netstandard1.3-Windows_NT;
```

Comment:
Yeah I was surprised to find that we don't actually have a tracking issue for this. We should be able to get this to work, by the way, it's just work to be done. There are packages for ilasm that should work everywhere we need them to.